Core/DolphinVM/strgprim.cpp

```diff
@@ -374,7 +374,7 @@ Oop* __fastcall Interpreter::primitiveStringNextIndexOfFromTo(Oop* const sp, pri
 									char32_t c;
 									auto next = i;
 									
-									U8_NEXT(s, next, to, c)
+									U8_NEXT(s, next, to, c);
```

Comment:
This showed up with the 10.22000 MS SDK release that I seemed to have responded to in July with ad9eb46f3e5b88d12166af7b958a4e646d5b1435 in master.

Reply:
Good to have in in 7.1 then as well.